pkg/lightning/backend/local/region_job.go

```diff
@@ -780,6 +780,16 @@ func (local *Backend) doIngest(ctx context.Context, j *regionJob) (*sst.IngestRe
 	return resp, nil
 }
 
+// UpdateLimiter updates the write limiter of the backend.
+func (local *Backend) UpdateLimiter(limit int) {
+	local.writeLimiter.UpdateLimiter(limit)
+}
+
+// GetLimiterSpeed returns the speed of the write limiter.
+func (local *Backend) GetLimiterSpeed() int {
```

Comment:
write a unit test to check they are concurrent-safe

Reply:
Do you mean like this https://github.com/pingcap/tidb/pull/57611/commits/8c07af02cd89bdf8311245955f3ac8759bca4eda ?